src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -57,6 +57,7 @@ class DataInterfaceTestMixin:
     data_interface_cls: Type[BaseDataInterface]
     interface_kwargs: Union[dict, List[dict]]
     save_directory: Path = Path(tempfile.mkdtemp())
+    maxDiff = None
```

Comment:
What is going on here? I don't see `maxDiff` anywhere else.

Reply:
I use this a fair amount in the NWB Inspector I believe; it's useful when running `pytest -vv` (extra verbose) because it doesn't truncate output.

Also useful for the GitHub action logs so it doesn't truncate the report for long message returns